tests/transports-governance/test_governance_hierarchical_validation.py

```
@@ -0,0 +1,895 @@
+        track_created_entity("customers", customer_id)
+
+        team_data = test_data_factory.team_data(
+            customer_id=customer_id, budget_id=team_budget_id
+        )
+        team_response = governance_client.create_team(**team_data)
+        team_id = team_response.json()["id"]
+        track_created_entity("teams", team_id)
+
+        # Try to reduce customer budget below team budget
+        invalid_update_response = governance_client.update_budget(
+            customer_budget_id, {"max_limit": 3000}  # < 5000 (team budget)
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)